File: src/main/java/com/github/svegon/capi/mixininterface/IMerchantScreenHandler.java
package com.github.svegon.capi.mixininterface;

import net.minecraft.village.Merchant;
import net.minecraft.village.MerchantInventory;

public interface IMerchantScreenHandler {
    Merchant getMerchant();

    void setMerchant(Merchant merchant);

    MerchantInventory getMerchantInventory();

    void setMerchantInventory(MerchantInventory merchantInventory);

    void playYesSoundServer();
}

File: src/main/kotlin/io/github/svegon/mclientapi/mixininterface/IMerchantScreenHandler.kt
package io.github.svegon.mclientapi.mixininterface

import net.minecraft.village.Merchant
import net.minecraft.village.MerchantInventory

interface IMerchantScreenHandler {
    val merchant: Merchant

    val merchantInventory: MerchantInventory

    fun playYesSoundAsServer()
}
